### DISCUSSION TITLE
Can you useQueries with initialData?

### DISCUSSION THREAD

**Discussion by londonadagio** [TanStack/query#4907] [Q&A] [answered]
Hey everyone!

Just a simple question, I need to run multiple parallel queries in a NextJS component, useQueries works great but I can't find a way to provide initialData for it. Am I missing something?

In my use case I can just call useQuery multiple times instead, but I'm just curious in case I run into a situation where I need to run useQueries.

**Comment by louis-young** [comment, marked as answer]
Hi 👋 

You should be able to, yes. The `useQueries` query options are almost identical to `useQuery`.

Please see: https://tanstack.com/query/v4/docs/react/reference/useQueries

If this isn't working for you, please provide a runnable minimal reproducible example so we can take a look 🙂
